CONTRIBUTING.md

```diff
@@ -423,7 +423,7 @@ If you're new to Bevy, here's the workflow we use:
 2. Make your changes in a local clone of your fork, typically in its own new branch.
    1. Try to split your work into separate commits, each with a distinct purpose. Be particularly mindful of this when responding to reviews so it's easy to see what's changed.
    2. Tip: [You can set up a global `.gitignore` file](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) to exclude your operating system/text editor's special/temporary files. (e.g. `.DS_Store`, `thumbs.db`, `*~`, `*.swp` or `*.swo`) This allows us to keep the `.gitignore` file in the repo uncluttered.
-3. To test CI validations locally, run the `cargo run -p ci` command. This will run most checks that happen in CI, but can take some time. You can also run sub-commands to iterate faster depending on what you're contributing:
+3. To test CI validations locally, run the `cargo run -p ci` command. This will run most checks that happen in CI, but can take some time. Some systems may experience [linker performance issues when running doc tests](https://github.com/bevyengine/bevy/issues/12207). You can also run sub-commands to iterate faster depending on what you're contributing:
```

Comment:
We're slowly moving information from here to the [Contributing Guide](https://bevyengine.org/contributing/introduction/). I'm fine with this being added for now, but once https://github.com/bevyengine/bevy-website/pull/1605 is merged we should copy this information over.

Reply:
Worth noting this may be fixed if [rust-lld](https://blog.rust-lang.org/2024/05/17/enabling-rust-lld-on-linux.html) makes it into 1.82/rust 2024; this issue no longer occurs on nightly.